connector/.env

```diff
@@ -38,16 +25,18 @@ WEB_HTTP_PROTOCOL_PORT=11003
 WEB_HTTP_CONTROL_PORT=11004
 WEB_HTTP_PATH=${MY_EDC_BASE_PATH}/api
 WEB_HTTP_MANAGEMENT_PATH=${MY_EDC_BASE_PATH}/api/v1/management
-WEB_HTTP_PROTOCOL_PATH=${MY_EDC_BASE_PATH}/api/v1/ids
+WEB_HTTP_PROTOCOL_PATH=${MY_EDC_BASE_PATH}/api/v1/dsp
```

Comment:
Consistency: if we go for /dsp, we need to switch it everywhere we now documented /protocol